bot/resources/tags/dict-get.md

```diff
@@ -0,0 +1,15 @@
+Often while using dictionaries in Python, you may run into `KeyErrors`. This error is raised when you try to access a key that isn't present in your dictionary.\
+While you can use a `try` and `except` block to catch the `KeyError`, Python also gives you some other neat ways to handle them.
```

Comment:
```suggestion
While using dictionaries in Python, you may run into `KeyError`s. This is raised when you try to access a dictionary item that isn't present in the dictionary. Python gives you some neat ways to handle them.
```
This could be condensed and more clear.

Reply:
I was told to mention the `try - except` option in a single line here, do you think I should omit it?

Reply:
If you want to do so, include it. I personally think it's a bit redundant.